mlflow/h2o.py

```diff
@@ -60,6 +69,21 @@ def log_model(h2o_model, artifact_path, **kwargs):
     :param h2o_model: H2O model to be saved.
     :param artifact_path: Run-relative artifact path.
     :param kwargs: kwargs to pass to ``h2o.save_model`` method.
+
+    >>> import mlflow
+    >>> import mlflow.h2o
+    >>> import h2o
+    >>> from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+    >>> #Partial example code used from h2o documentation <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/early_stopping.html>
+    >>> # Initialize and train a GLM
+    >>> h2o.init()
+    >>> h2o_model = H2OGeneralizedLinearEstimator(family = 'binomial', early_stopping = True)
+    >>> h2o_model.train(...)
+    >>> #log parameters
+    >>> mlflow.log_param("early_stopping", "True")
```

Comment:
I think we can remove the log param calls.

Reply:
I suppose we could remove it. Will do,